### ISSUE TITLE
Small bug related to special letters: я -> ya

### ISSUE THREAD

**Issue by abduakhatov** [barseghyanartur/transliterate#43]
When translated from latin the text to cyrillic-russian. Example translated -
using lib: (`"ya neznayu" -> "ыа незнаыу"`)
supposed to be: ("`ya neznayu" -> "я незнаю`")

**Comment by skoval00** [comment]
This is expected behaviour. Right now project does not have manpower to fix similar problems(#12) since it requires some redesign. You can apply pull-request #13 for yourself and add any conversions you like as a temporary workaround.

**Comment by barseghyanartur** [comment]
Pull request #13 was almost ok, besides that some tests were failing. It's the right approach to broaden the ``pre_processor_mapping`` and ``reversed_pre_processor_mapping`` for such cases. Help here is highly appreciated.

**Comment by abduakhatov** [comment]
Yeah, for sure bro, sooner I will be free from my project, so except the pull request regarding the bug mentioned, I am going to add languages from two different language families. Still, thank you for this project
